### DISCUSSION TITLE
How do I use .info/.civitai.info files?

### DISCUSSION THREAD

**Discussion by schumar** [vladmandic/sdnext#1629] [Q&A]
What needs to be put in `.info`/`.civitai.info` files? (see 8000d19)

Is there some tool to download that info? (I guess I could whip one up, but that would require knowing what info SD.next expects to be in there)

**Comment by brunogcar** [comment]
if im not mistaken https://github.com/butaixianran/Stable-Diffusion-Webui-Civitai-Helper 

grabs civitai details  for each models,  version,  pics,  etc and create that info file

**Reply by schumar** [reply]
I think that project is not actively maintained anymore, and doesn't work with the current SD.next version :(
(see #1557)
There is a [fork by goldmojo](https://github.com/goldmojo/Stable-Diffusion-Webui-Civitai-Helper) which has 3 more commits, but also doesn't work.

**Comment by schumar** [comment]
Okay, I noticed that I had 1 `.civitai.info` file lying around, from when I last used butaixianran's extension.

It seems that SD.next just takes the content of the file and displays it as-is when I click the little `i`-icon.

I was somehow hoping for it to actually parse the JSON -- I would still *love* to have something like the ideas thrown around in [sdwebui \#8029](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/8029) in SD.next.

**Reply by vladmandic** [reply]
eventually when sdnext creates a built-in civitai integration instead of relying on 3rd party extensions. then i can guarantee the format. until then, i don't want to parse someone else's format.

**Comment by schumar** [comment]
In case anyone is looking for a working script to automatically download preview images, and create `description.txt`/`.info` files, 
I've hacked together [a little something](https://gitlab.com/schumar/sdhelpers).
